code/game/turfs/simulated/floor/asteroid.dm

```diff
@@ -0,0 +1,317 @@
+/**********************Asteroid**************************/
+
+/turf/simulated/floor/plating/asteroid
```

Comment:
The turfs are the thing that are out of date with /tg/, so in the future we should make some improvements to them. They're perfectly functional right now though.